src/app/portlets/dot-templates/dot-template/dot-template.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="content-wrapper">
+    <form class="p-fluid" [formGroup]="group" (ngSubmit)="onSubmit($event)">
+        <button pButton label="Save" [disabled]="isDisabled" class="p-button-secondary"></button>
+        <div class="p-field">
+            <label for="title">Title</label>
+            <input id="title" formControlName="title" type="text" placeholder="Title" pInputText />
+            <dot-field-validation-message
+                [field]="group.controls['title']"
+                message="This field is required"
+            >
+            </dot-field-validation-message>
+        </div>
+        <!-- /.p-field -->
+        <div class="p-field">
+            <label for="description">Description</label>
+            <input
+                id="description"
+                formControlName="friendlyName"
+                type="text"
+                placeholder="Description"
+                pInputText
+                required
+            />
+        </div>
+        <!-- /.p-field -->
+        <div class="p-field">
+            <label for="button">Template</label> <br />
+            <dot-container-selector (change)="containerChange($event)"> </dot-container-selector>
+            <dot-textarea-content
+                #value
+                height="30rem"
+                formControlName="body"
+                required
+                [show]="['code']"
+                (monacoInit)="initEditor($event)"
+            ></dot-textarea-content>
+            <dot-field-validation-message
+                [field]="group.controls['body']"
+                message="This field is required"
```

Comment:
https://github.com/dotCMS/core-web/pull/1497/files#r517397880

Reply:
On hold for now.